Firmware/Marlin_main.cpp

```diff
@@ -8581,24 +8581,44 @@ SERIAL_PROTOCOLPGM("\n\n");
     } break;
 
     /*!
-    ### M709 - MMU reset <a href="https://reprap.org/wiki/G-code#M709:_MMU_reset">M709: MMU reset</a>
-    The MK3S cannot not power off the MMU, for that reason the functionality is not supported.
+    ### M709 - MMU power & reset <a href="https://reprap.org/wiki/G-code#M709:_MMU_power_&_reset">M709: MMU power & reset</a>
+    The MK3S cannot not power off the MMU, but we can en- and disable the MMU.
     #### Usage
 
-        M709 [ X ]
+        M709 [ S | X ]
 
     #### Parameters
-    - `X` - Reset MMU (0:soft reset | 1:hardware reset)
+    - `X` - Reset MMU (0:soft reset | 1:hardware reset | 42: erease MMU eeprom)
```

Comment:
erease => erase

Reply:
Also, it should be noted, that the new state of the MMU is stored in printer's EEPROM

Reply:
@gudnimg fixed typo and @DRracer updated documentation

Reply:
@3d-gussner : with the EEPROM comment I meant something like this:
"The new state of the MMU is stored in printer's EEPROM - i.e. if you disable the MMU via M709, it will not be activated after the printer resets."

Reply:
@DRracer Better now? See https://github.com/prusa3d/Prusa-Firmware/pull/4474/commits/5cdb1916102fbac9bd5bace0509367911a6175e7